e2e/e2e_test.go

```diff
@@ -261,6 +267,19 @@ func TestE2E(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "jwt-port",
+			verify: []verifyCmd{
+				{
+					from: "sleep.foo", to: "httpbin.foo:80", path: "/headers",
```

Comment:
could you change to use port "8000", here and all other places?

This is to match the httpbin deploy in https://github.com/yangminzhu/security-policy-migrate/blob/463a61bf44e748c57790d9c2c72395b73c6d003a/e2e/deploy/httpbin.yaml#L32-L33 where it now omits the targetPort, then the targetPort will be the same as the service port (8000).